docs/ReleaseNotes.md

```diff
@@ -7,7 +7,7 @@ Acuminator 3.0 includes the bug fixes and enhancements described in this section
 ### Enhancements
 In Acuminator 3.0, the following enhancements have been implemented:
  - Acuminator now supports Visual Studio 2022 in addition to the two previous versions of Visual Studio, 2017 and 2019. Also, Acuminator now supports preview versions of Visual Studio. 
- - The [PX1007](diagnostics/PX1007.md) diagnostic now checks only DACs and DAC extensions.
+ - The [PX1007](diagnostics/PX1007.md) diagnostic now checks only DACs, DAC extensions and their public DAC field properties.
```

Comment:
Please change to
checks only DACs, DAC extensions, and public field properties for these classes.

Reply:
Done